packages/blocks/src/ranking-question/index.js

```diff
@@ -38,7 +38,7 @@ const RankingQuestion = ( { attributes, children, className } ) => {
 
 		const [ removed ] = answers.splice( source.index, 1 );
 		answers.splice( destination.index, 0, removed );
-		onSort( answers );
+		onUpdate( map( answers, 'props.attributes.clientId' ) );
```

Comment:
Note: Since this particular use case is not relying on any inputs - this totally make sense here :)